internal/controller/platform/services/services.go

```diff
@@ -603,6 +603,7 @@ func (d *DataIndexHandler) GenerateKnativeResources(platform *operatorapi.Sonata
 		d.newTrigger(lbl, brokerName, namespace, serviceName, "process-state", "ProcessInstanceStateDataEvent", constants.KogitoProcessInstancesEventsPath, platform),
 		d.newTrigger(lbl, brokerName, namespace, serviceName, "process-variable", "ProcessInstanceVariableDataEvent", constants.KogitoProcessInstancesEventsPath, platform),
 		d.newTrigger(lbl, brokerName, namespace, serviceName, "process-definition", "ProcessDefinitionEvent", constants.KogitoProcessDefinitionsEventsPath, platform),
+		d.newTrigger(lbl, brokerName, namespace, serviceName, "process-instance-multiple", "MultipleProcessInstanceDataEvent", constants.KogitoProcessInstancesMultiEventsPath, platform),
```

Comment:
What happens if I toggle to `false` and then to `true` again? Are we recreating/deleting the trigger instance?

Reply:
This basically adds a new trigger, with the same pattern for the other existing ones.

@jianrongzhang89 do you know what happens in the situation introduced by @ricardozanini ?

Reply:
@ricardozanini current implementation does not delete the existing triggers if it the flag is set from true to false (and they will get deleted if the CR is deleted) and when it is to true again - the new trigger gets added.

Reply:
So we will have dangling triggers in case we turn it to false?

Reply:
Yes.